### PR TITLE
Document why you can't programmatically update UI from JS output

### PR DESCRIPTION
**Pull request by saulshanabrook**
## References

Addresses https://github.com/jupyterlab/jupyterlab/issues/4623

## Code changes

None
## User-facing changes

Adds section to FAQ to document why you can't run JS from a notebook output that changes the UI.

### CODE REVIEW

File: docs/source/getting_started/faq.rst [hunk 1]
```diff
@@ -21,6 +21,17 @@ Development
 -  `How can you
    contribute? <https://github.com/jupyterlab/jupyterlab/blob/master/CONTRIBUTING.md>`__
 -  :ref:`How can you extend or customize JupyterLab? <user_extensions>`
+-  In the classic Notebook, `I could use custom Javascript outputed by a cell to programatically
+   control the Notebook <https://stackoverflow.com/a/32769976/907060>`__. Can I do the same thing in JupyterLab?
+
+   JupyterLab was built to support a wide variety of extensibility, including dynamic behavior based on notebook
+   outputs.To access this extensability, you should write a custom JuptyerLab extension. If you would
```
Comment (ivanov) [inline docs/source/getting_started/faq.rst]: missing space after period, before "To access"

Comment (saulshanabrook) [inline docs/source/getting_started/faq.rst]:
```suggestion
   outputs. To access this extensability, you should write a custom JuptyerLab extension. If you would
```

Comment (saulshanabrook) [reply docs/source/getting_started/faq.rst]: Thanks!

File: docs/source/getting_started/faq.rst [hunk 2]
```diff
@@ -21,6 +21,17 @@ Development
 -  `How can you
    contribute? <https://github.com/jupyterlab/jupyterlab/blob/master/CONTRIBUTING.md>`__
 -  :ref:`How can you extend or customize JupyterLab? <user_extensions>`
+-  In the classic Notebook, `I could use custom Javascript outputed by a cell to programatically
+   control the Notebook <https://stackoverflow.com/a/32769976/907060>`__. Can I do the same thing in JupyterLab?
+
+   JupyterLab was built to support a wide variety of extensibility, including dynamic behavior based on notebook
+   outputs. To access this extensability, you should write a custom JuptyerLab extension. If you would
```
Comment (jtpio) [inline docs/source/getting_started/faq.rst]: Noticed another quick one:

```suggestion
   outputs. To access this extensibility, you should write a custom JupyterLab extension. If you would
```

Comment (saulshanabrook) [reply docs/source/getting_started/faq.rst]: Thanks!

**Pull request merged**